data/content/waypoint/country/MW.cup

```diff
@@ -1,10 +1,81 @@
+name,code,country,,,elev,style,rwydir,rwylen,freq,desc
+Chileka NDB,CL,MW,1540.833S,03457.483E,807.15m,10,,,,253khz Coverage 200NM
+Lilongwe VOR,DVL,MW,1348.672S,03348.471E,1218.83m,9,,,114.9,114.9Mhz Coverage 200NM
+Karonga NDB,KA,MW,0957.100S,03353.700E,512.99m,10,,,,245khz Coverage 50NM
+Lilongwe NDB,LH,MW,1350.550S,03350.850E,1200.08m,10,,,,378khz Coverage 200NM
+Mzuzu NDB,UU,MW,1126.583S,03400.833E,1277.95m,10,,,,227khz Coverage 50NM
+Chileka VOR,VCL,MW,1540.365S,03454.992E,687.94m,9,,,113.3,113.3Mhz Coverage 200NM
+Karonga VOR,VKA,MW,0957.239S,03353.472E,512.99m,9,,,115.3,115.3Mhz Coverage 100NM
+Mzuzu VOR,VMZ,MW,1124.817S,03400.108E,1295.01m,9,,,116.1,116.1Mhz Coverage 100NM
+ANTOP,ANTOP,MW,1130.000S,03338.500E,,1,,,,Reporting Point
+AXEBO,AXEBO,MW,1331.767S,03249.650E,,1,,,,Reporting Point
+ELEPA,ELEPA,MW,1456.667S,03304.583E,,1,,,,Reporting Point
+ESPOP,ESPOP,MW,1500.000S,03316.000E,,1,,,,Reporting Point
+GIPVO,GIPVO,MW,1140.000S,03318.000E,,1,,,,Reporting Point
+IMKAL,IMKAL,MW,1123.500S,03425.750E,,1,,,,Reporting Point
+KUSKI,KUSKI,MW,1325.300S,03452.000E,,1,,,,Reporting Point
+LABON,LABON,MW,1340.000S,03248.000E,,1,,,,Reporting Point
+LONON,LONON,MW,1338.000S,03501.500E,,1,,,,Reporting Point
+MEKRO,MEKRO,MW,1042.000S,03335.000E,,1,,,,Reporting Point
+MIROS,MIROS,MW,1538.000S,03403.000E,,1,,,,Reporting Point
+MUTAR,MUTAR,MW,1500.000S,03343.000E,,1,,,,Reporting Point
+NIBOR,NIBOR,MW,1236.000S,03445.667E,,1,,,,Reporting Point
+OKMAT,OKMAT,MW,1618.283S,03420.400E,,1,,,,Reporting Point
+ORLIM,ORLIM,MW,1048.000S,03425.000E,,1,,,,Reporting Point
+OVANA,OVANA,MW,0936.833S,03330.000E,,1,,,,Reporting Point
+TIBAK,TIBAK,MW,1244.000S,03259.417E,,1,,,,Reporting Point
+UTALA,UTALA,MW,1125.000S,03337.667E,,1,,,,Reporting Point
+KAVAL,KAVAL,MW,1532.283S,03358.200E,,1,,,,Reporting Point WGS84
+LABLO,LABLO,MW,1557.583S,03406.500E,,1,,,,Reporting Point WGS84
+MELNA,MELNA,MW,1551.883S,03545.400E,,1,,,,Reporting Point WGS84
+NALSA,NALSA,MW,1259.167S,03352.917E,,1,,,,Reporting Point WGS84
+TETEM,TETEM,MW,1443.033S,03420.667E,,1,,,,Reporting Point WGS84
+Bangula MBC,,MW,1634.650S,03506.383E,,8,,,,"Light HBN, FLG R, HN"
+Chilumba,,MW,1025.000S,03416.000E,,8,,,,"Light Marine, FLG(1) W EV 9 Sec, H24"
+Chizumulu Island,,MW,1201.000S,03437.000E,,8,,,,"Light Marine, FLG(1) W EV 9 Sec, H24"
+Kambwe Lagoon,,MW,0953.000S,03357.000E,,8,,,,"Light Marine, FLG(1/2) W EV 4.5 Sec, H24"
+Karonga MBC,,MW,0956.783S,03354.917E,,8,,,,"Light HBN, FLG R, HN"
+Katenga Isle,,MW,1355.000S,03438.000E,,8,,,,"Light Marine, FLG(1) W EV 9 Sec, H24"
+Lilongwe MBC,,MW,1400.117S,03345.100E,,8,,,,"Light HBN, FLG R, HN"
+Mangochi MBC,,MW,1427.217S,03513.567E,,8,,,,"Light HBN, FLG R, HN"
+Mbenje Isle,,MW,1325.000S,03429.000E,,8,,,,"Light Marine, FLG(1/4) W EV 2.25 Sec, H24"
+Mzuzu MBC,,MW,1130.000S,03356.000E,,8,,,,"Light HBN, FLG R, HN"
+Namalenje Island,,MW,1344.000S,03438.000E,,8,,,,"Light Marine, FLG(1/2) W EV 4.5 Sec, H24"
+Ngumbe MBC,,MW,1542.083S,0351.650E,,8,,,,"Light HBN, FLG R, HN"

```

Comment:
The longitude here is missing a digit. "0351.650E"

Reply:
Done. Hopefully it works now.